app/views/shared/_navbar.html.erb

```diff
@@ -1,59 +1,63 @@
 <div class="navbar-wagon navbar-wagon-fixed">
-  <!-- Logo -->
-  <a href="/" class="navbar-wagon-brand">
-    <%= image_tag "unpolarise.svg" %>
-  </a>
+
+  <!-- Left Navigation -->
+  <div class="navbar-wagon-left hidden-xs hidden-sm">
+    <!-- Logo -->
+    <%= link_to root_path, class: "navbar-wagon-brand" do %>
+      <%= image_tag "unpolarise.svg" %>
+    <% end %>
+    <!-- Search form: The search form appears only if the user is logged in -->
+    <% if user_signed_in? %>
+      <form action="" class="navbar-wagon-search">
+        <input type="text" class="navbar-wagon-search-input" placeholder="What are you interested in?">
+        <button type="submit" class="navbar-wagon-search-btn">
+          <i class="fa fa-search"></i>
+        </button>
+      </form>
+    <% end %>
+  </div>
 
   <!-- Right Navigation -->
   <div class="navbar-wagon-right hidden-xs hidden-sm">
+    <% if user_signed_in? %>
+      <!-- Link to Articles owned by the user -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Articles</a>
```

Comment:
Here, we can add the link to the user bookings:
<%= link_to "Articles", user_bookings_path(current_user), class: "navbar-wagon-item navbar-wagon-link" %>
We also need to change the comment to "Link to Articles booked by the user"